build.ps1

```diff
@@ -43,7 +43,7 @@ function CrossGen([string] $runtime, [string] $publishTarget, [string] $privateS
     dotnet publish .\src\WebJobs.Script.WebHost\WebJobs.Script.WebHost.csproj -r $runtime -o "$selfContained" -v q /p:BuildNumber=$buildNumber    
 
     # Modify web.config for inproc
-    dotnet tool install -g dotnet-xdt --version 2.1.0-rc.1
+    dotnet tool install -g dotnet-xdt --version 2.1.0 2> $null
```

Comment:
We could move this elsewhere, but for now, I'm just sending stderr nowhere. Also updated to 2.1.0